Tiltfile

```diff
@@ -212,10 +212,10 @@ def capz():
     yaml = str(kustomizesub("./hack/observability"))  # build an observable kind deployment by default
 
     # add extra_args if they are defined
-    if settings.get("extra_args"):
-        azure_extra_args = settings.get("extra_args").get("azure")
+    if settings.get("container_args"):
+        capz_container_args = settings.get("container_args").get("capz-controller-manager")
```

Comment:
We have often struggled in getting more logs outputted while running in tilt. 
With this change, if we add the below yaml fields to our tilt-settings.yaml, we will can play around with more logging
```yaml
container_args:
  capz-controller-manager:
    - "--v=2"

```